Code/Engine/GameEngine/Effects/Shake/Implementation/CameraShakeComponent.cpp

```diff
@@ -67,7 +67,7 @@ void ezCameraShakeComponent::GenerateKeyframe()
 
   if (deviation > ezAngle())
   {
-    m_Rotation += ezAngle::Radian(pWorld->GetRandomNumberGenerator().DoubleMinMax(ezAngle::Degree(120).GetRadian(), ezAngle::Degree(240).GetRadian()));
```

Comment:
I just relocated this file in my PR, so I have integrated your fix there. In case my PR goes in first, you can just remove this change from your PR.

Reply:
Ok, I'll removed this change then.